.github/workflows/build.yml

```diff
@@ -299,16 +299,17 @@ jobs:
                 2> >(tee build.log >&2) || true
             fi 
 
-            # If build did not fail due to broken pipe, break and continue
-            if ! grep -q "client_loop: send disconnect: Broken pipe" build.log
+            # If build did not fail due to broken pipe, break and continue.
+            # If build.json exits, the build compleated.
```

Comment:
this is either [mtg](https://mtg.fandom.com/wiki/Compleation) or a spelling error :wink: